ci/azure-jobs-linux.yml

```diff
@@ -57,3 +57,35 @@ jobs:
       UPLOAD_USER: $(ARTIFACTS_UPLOAD_USER)
       UPLOAD_PASS: $(ARTIFACTS_UPLOAD_PASS)
       UPLOAD_SIGN_KEY: $(ARTIFACTS_UPLOAD_SIGN_KEY)
+- job: Linux_Unbundled
```

Comment:
That might be worth a try. I don't remember why I split up the steps (probably because I overlooked that the appimage and archive are only built conditionally).

Reply:
Good thing you suggested that, the libraries were not actually being installed by `make install`. It should now be fixed. I also merged the jobs by adding some conditionals.